### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Mange99**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,18 +1,24 @@
-import { HeaderComponent } from "./components/header/headerComponent";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
-import { ExercisePage } from "../src/components/exercisePage/exercisePage";
-import { CalculatorPage } from "../src/components/calculatorPage/calculatorPage";
+import { ExercisePage } from "./pages/exercisePage";
+import { CalculatorPage } from "./pages/calculatorPage";
+import LandingPage from "./pages/LandingPage";
+import { HeaderComponent } from "./components/header/headerComponent";
+import { Box, Container } from "@chakra-ui/react";
```
Comment (Torbi) [inline src/App.tsx]: Remove unused imports

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      w="full"
+      h={80}
+      pt={16}
+      mt={16}
+      backgroundColor="black"
+      textAlign="center"
+    >
+      <Box w="full" h="3px" backgroundColor="white"></Box>
+      <Heading textColor="white" pt={4}>
+        inspirational quote
```
Comment (Torbi) [inline src/components/Footer.tsx]: meby actual inspirational quote. Sic vis paces, para bellum

File: src/pages/Hero.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import { Box, Button, Heading, Icon, Image } from "@chakra-ui/react";
+import bg from "../res/images/hero.jpg";
+import { MdArrowDownward } from "react-icons/md";
+
+interface HeroProps {
+  scrollToNext: () => void;
+}
+
+const Hero = ({ scrollToNext }: HeroProps) => {
```
Comment (Torbi) [inline src/pages/Hero.tsx:9]: Why is it named Hero? Does not feel very descriptive

File: src/pages/LandingPage.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+import SummarizeCard from "../components/summarizeCard";
+import { BmiCalculator } from "../components/calculatorCards/bmiCalculator";
+
+const LandingPage = () => {
+  const scrollToNext = () => {
+    workoutsRef.current.scrollIntoView({ behavior: "smooth" });
+  };
+
+  const workoutsRef =
+    useRef<HTMLDivElement>() as MutableRefObject<HTMLDivElement>;
```
Comment (Torbi) [inline src/pages/LandingPage.tsx:28]: why not on a single row?

File: src/pages/LandingPage.tsx [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+          </Link>{" "}
+        </Box>
+        <Heading pt={24} fontWeight="bold" textAlign="center" size="xl">
+          Workouts
+        </Heading>
+        <Grid templateColumns="repeat(3, 1fr)" gap={6} mt={8}>
+          <GridItem w="80%" margin="auto" height="100%">
+            <SummarizeCard
+              image={Excercise}
+              title="Choose from 1000+ excercises"
```
Comment (Torbi) [inline src/pages/LandingPage.tsx]: spelling

File: src/pages/LandingPage.tsx [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+              description="With Fitness you can find ower 1000 different workouts and star the excersices you like the most."
```
Comment (Torbi) [inline src/pages/LandingPage.tsx]: spelling

File: src/pages/LandingPage.tsx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+            />
+          </GridItem>
+          <GridItem w="80%" margin="auto" height="100%">
+            <SummarizeCard
+              image={Filter}
+              title="Filter"
+              description="You can filter excercises for different body parts or equipment you have access to and much more!"
```
Comment (Torbi) [inline src/pages/LandingPage.tsx]: spelling

File: src/pages/LandingPage.tsx [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+            />
+          </GridItem>
+          <GridItem w="80%" margin="auto" height="100%">
+            <SummarizeCard
+              image={Excercise}
```
Comment (Torbi) [inline src/pages/LandingPage.tsx]: spelling

File: src/pages/LandingPage.tsx [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+              title="Create your own sessions"
+              description="You can create your own workout passes with different workouts to help you thorugh your journey"
```
Comment (Torbi) [inline src/pages/LandingPage.tsx]: sessions instead of passes? feels like better grammar

**Review by Torbi — CHANGES_REQUESTED** [review]
Good, mostly spelling errors and some general questions about stuff

File: src/components/Footer.tsx (hunk 2 above)
Comment (Mange99) [reply src/components/Footer.tsx]: Sī vīs pācem, parā bellum*

File: src/pages/Hero.tsx (hunk 3 above)
Comment (Mange99) [reply src/pages/Hero.tsx:9]: A hero image is a website design term used to describe an oversized banner image at the top of a website.

**Review by Torbi — APPROVED** [review]
lgtm

**Pull request merged**